isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp

```diff
@@ -75,18 +75,26 @@ namespace Isis {
     // The observation start time and clock count for SRC are based on the center of the exposure.
     Pvl &lab = *cube.label();
     PvlGroup &inst = lab.findGroup("Instrument", Pvl::Traverse);
-    QString clockCount = inst["SpacecraftClockStartCount"];
-    double et = getClockTime(clockCount).Et();
-    double exposureDuration = (double)inst["ExposureDuration"] / 1000.0;
 
-    pair<iTime, iTime> startStop = ShutterOpenCloseTimes(et, exposureDuration);
-    setTime(et);
+    // scs 2022-04-16 Changed to use the StartTime instead of the StartClock.
+    // HRSC SRC images taken in a sequence (e.g., Phobos) often have identical StartClock values,
+    // but different StartTime values
+    //QString clockCount = inst["SpacecraftClockStartCount"];
+    //double et = getClockTime(clockCount).Et();
+    //double exposureDuration = (double)inst["ExposureDuration"] / 1000.0;
```

Comment:
Do we still need exposureDuration?

Reply:
It's not used anywhere else, so it is ok to remove